.github/workflows/dep-update-review.yml

```diff
@@ -0,0 +1,72 @@
+name: Dependency Update Reviewer
```

Comment:
I think it might be better to include this job in the same workflow file as the other dependency notifier job.

Reply:
does it mean to move this job to `dep-summaries.yml` ?

If that is the case, one key difference is the path filtering based on manifest files. We only run this job when the diff contains any `Cargo.lock` or `Cargo.toml` file.

Reply:
that's a good point, i suppose we can leave this as its own file for now then